libcxx/test/std/containers/sequences/vector.bool/move.pass.cpp

```diff
@@ -9,54 +9,70 @@
 // UNSUPPORTED: c++03
 
 // <vector>
+// vector<bool>
 
 // vector(vector&& c);
 
-#include <vector>
+#include <array>
 #include <cassert>
-#include "test_macros.h"
-#include "test_allocator.h"
+#include <vector>
+
 #include "min_allocator.h"
+#include "test_allocator.h"
+#include "test_macros.h"
+
+template <unsigned N, class A>
+TEST_CONSTEXPR_CXX20 void test(const A& a) {
+  std::vector<bool, A> v(N, false, a);
+  std::vector<bool, A> v0(N, false, a);
```

Comment:
```suggestion
  std::vector<bool, A> copy(N, false, a);
```

I think the test would be more readable like that. Or `std::vector<bool, A> original`.